src/client/package.json

```diff
@@ -6,5 +6,9 @@
     "gulp": "^3.8.10",
     "gulp-watch": "^3.0.0"
   },
+  "scripts": {
+    "build": "gulp",
```

Comment:
Maybe rename this to 
```"gulp": "gulp",```
as this allows us to use gulp for other gulp tasks too.
```npm gulp <task>```